unifi_respondd/config.py

```diff
@@ -73,6 +73,7 @@ def from_dict(cls, cfg: Dict[str, str]) -> "Config":
             unicast_port=cfg["unicast_port"],
             interface=cfg["interface"],
             verbose=cfg["verbose"],
+            fallback_domain=cfg.get("fallback_domain", None),
```

Comment:
maybe change `None` to `unifi_respondd_fallback`

Reply:
needs to be added also somewhere here:
https://github.com/grische/unifi_respondd/blob/feature/fallback-domain-configurable/unifi_respondd/config.py#L31-L45

Reply:
https://github.com/freifunkMUC/omada_respondd/commit/0a421543e7e6d20db505a58e366076b5a63a8e76

Reply:
@T0biii updated

Reply:
Are you sure about `unifi_respondd_fallback`? Currently, it's `ffmuc_unifi_respondd_fallback` .

Reply:
as [omada](https://github.com/freifunkMUC/omada_respondd) and the [uisp_respondd](https://github.com/freifunkMUC/uisp_respondd ) uses also only x_respondd_fallback i think it would be the best
anyone can ajust it as they want :)